components/Filter.js

```diff
@@ -8,81 +8,67 @@ import {
   SiNewbalance,
 } from "react-icons/si";
 import { GiConverseShoe } from "react-icons/gi";
-import { useState } from "react";
-
-export default function Filter() {
-  const [selectedIcon, setSelectedIcon] = useState();
-  const [isActive, setIsActive] = useState(false);
-
-  function handleToggleIcon() {
-    setIsActive(!isActive);
-  }
 
+export default function Filter({ selectedIcon, setSelectedIcon }) {
   return (
     <>
       <StyledContentContainer>
         <StyledIconContainer>
-          <StyledBrandCircle onClick={() => handleToggleIcon()}>
+          <StyledBrandCircle>
+            <StyledBrandAll
+              onClick={() => setSelectedIcon("all")}
+              className={selectedIcon === "all" ? "clicked" : ""}
+            >
+              All Sneakers
+            </StyledBrandAll>
+          </StyledBrandCircle>
+          <StyledBrandCircle>
             <StyledBrandNike
               size="70px"
               onClick={() => setSelectedIcon("nike")}
```

Comment:
hier (und in allen Brands darunter) würde dann deine Funktion aufgerufen werden, die die Setter-Funktion ersetzt

Reply:
bereits umgestellt :)